tests/app/phase-tester-response/issue-pending/issue-pending.component.spec.ts

```diff
@@ -0,0 +1,53 @@
+import { IssuePendingComponent } from '../../../../src/app/phase-tester-response/issue-pending/issue-pending.component';
+import { Issue, STATUS } from '../../../../src/app/core/models/issue.model';
+import { ISSUE_WITH_EMPTY_DESCRIPTION } from '../../../constants/githubissue.constants';
+import { Team } from '../../../../src/app/core/models/team.model';
+import { IssueComment } from '../../../../src/app/core/models/comment.model';
+import { EMPTY_TEAM_RESPONSE, PENDING_TUTOR_MODERATION } from '../../../constants/githubcomment.constants';
+import { TutorModerationTodoTemplate } from '../../../../src/app/core/models/templates/tutor-moderation-todo-template.model';
+
+describe('IssuePendingComponent', () => {
+    describe('.ngOnInit()', () => {
+        const dummyTeam: Team = new Team({
+            id: 'dummyId',
+            teamMembers: [],
+          });
+        let dummyIssue: Issue;
+        let issuePendingComponent: IssuePendingComponent;
+        const issueComment: IssueComment = new TutorModerationTodoTemplate([PENDING_TUTOR_MODERATION]).comment;
```

Comment:
The issue comment being created here is from the moderation phase. Conceptually, it isn't right to use it for testing components under tester response phase (though the tests' logic is correct).

We can create the issue comment [as done in this PR](https://github.com/CATcher-org/CATcher/pull/599/files)

Reply:
I see, I didn't realise that! I will change it accordingly, thanks for pointing it out.